### PR TITLE
No iiot for makers

### PR DESCRIPTION
**Pull request by StefanKellerSiemens**
Still needs link to Marketplace.

With this pull request we have removed the complete documentation of the IIot for Makers case.
We have added a new chapter regarding the new Siemens Insights Hub API Authenticator and some details for the index.html patches.

### CODE REVIEW

File: content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    `https://{tenantName}.piam.{region}.{mindsphere-domain}/oauth/token?grant_type=client_credentials`
+
+    You have to replace `tenantName`, `region` and `mindsphere-domain` regarding your destination tenant e.g.
+
+    `https://demo.piam.eu1.mindsphere.io/oauth/token?grant_type=client_credentials`
+
+
+{{< figure src="/attachments/partners/siemens/mindsphere/mindsphere-api-only/ModuleConfiguration.png" alt="Authentication" >}}
+
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md:68]: Is this out of date. It seems to be repeating the information given in the paragraphs above.

File: content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+## 4 Authentication Considerations
+
+### 4.1 Authenticating During Development
+
+When you are developing your app, you can set the **ClientID** and **ClientSecret** constants within the app. You can also override these by using different [Configurations](/refguide/configuration/) within your app settings.
+
+{{% alert color="info" %}}
+**Tip:** Use the autofill feature based on a local environment variable for the *Client Secret*.
+
+Storing the *Client Secret* inside the app is, from a security perspective, not a good idea. A better approach is to use a local environment variable. Create a user-specific environment variable with *Variable name* equal to your *Client ID* value and the *Variable value* equal to your *Client Secret* value.
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md:82]: From Mendix 10.9 there is a solution to this where you can have a "Private" constant.
https://docs.mendix.com/refguide/configuration/#constants
But most customers will be using a version below 10.9, unfortunately.

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi Stefan,
Thanks for the PR.
I have proofread it in this commit: https://github.com/mendix/docs/pull/7557/commits/4e75b4f66b75a77cfd4b229d5a27d4aeb241f07b
And I have a couple of questions/comments.
Please can you let me know if everything is OK and I will mark this as ready for future release once the module is released in the Marketplace.
Yours
Mark

File: content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md (hunk 1 above)
Comment (StefanKellerSiemens) [reply content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md:68]: That is not out of date, it explain in more detail what needs to be provided for each constant.
But yes, it is a little bit duplicated and we could maybe align it to have it only at one place.

File: content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md (hunk 2 above)
Comment (StefanKellerSiemens) [reply content/en/docs/partners/siemens/mindsphere/mindsphere-api-only.md:82]: Good hint, we can maybe consider this for the next release. Right now we like to be compatible with the latest V9 LTS.

**Review by MarkvanMents — APPROVED** [review]
Thanks for the PR, Stefan.
Approving and will wait to hear from you about release date and link to the Marketplace component.

**Comment by MarkvanMents** [comment]
@StefanKellerSiemens -
This looks ready to go now.
I have one question: in the [deployment guide](https://docs.mendix.com/developerportal/deploy/deploying-to-mindsphere/) we have a reference to **Insights Hub IIot for Makers**.
image
Should we get rid of this as well, or do we need to keep it for now?

**Comment by StefanKellerSiemens** [comment]
Yes, please get rid of this.

**Comment by MarkvanMents** [comment]
Merge requested over email.

**Pull request merged**